test/evp_extra_test.c

```diff
@@ -2752,6 +2752,56 @@ static int test_RSA_get_set_params(void)
     return ret;
 }
 
+static int test_RSA_OAEP_set_get_params(void)
+{
+    int ret = 0;
+    EVP_PKEY *key = EVP_RSA_gen(2048);
+    EVP_PKEY_CTX *keyCtx = EVP_PKEY_CTX_new_from_pkey(0, key, 0);
+
+    if (nullprov != NULL)
+        return TEST_skip("Test does not support a non-default library context");
+
```

Comment:
Please replace with:

```suggestion
    EVP_PKEY *key = NULL;
    EVP_PKEY_CTX *key_ctx = NULL;
    
    if (nullprov != NULL)
        return TEST_skip("Test does not support a non-default library context");
    if (!TEST_ptr(key = load_example_rsa_key())
        || !TEST_ptr(key_ctx = EVP_PKEY_CTX_new_from_pkey(0, key, 0)))
        goto err;
```

Reply:
Thanks,  I should have noticed that :-)

Done.